tests/test_data/Sulfate_reducer_xgb_cccv.json

```diff
@@ -0,0 +1,728 @@
+		"stddev_balanced_accuracy": 0.02968179695191625,
+		"contamination": 0.3,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.8819642857142859,
+		"stddev_balanced_accuracy": 0.037101816459312334,
+		"contamination": 0.4,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.7261309523809525,
+		"stddev_balanced_accuracy": 0.04066238686077615,
+		"contamination": 0.5,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.670297619047619,
+		"stddev_balanced_accuracy": 0.08062662046394878,
+		"contamination": 0.6,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.589345238095238,
+		"stddev_balanced_accuracy": 0.07169454449085066,
+		"contamination": 0.7,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.581845238095238,
+		"stddev_balanced_accuracy": 0.056812038422772396,
+		"contamination": 0.8,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.49363095238095234,
+		"stddev_balanced_accuracy": 0.012757550785351249,
+		"contamination": 0.9,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.4655952380952382,
+		"stddev_balanced_accuracy": 0.044683949341997355,
+		"contamination": 1.0,
+		"completeness": 0.9
+	},
+	{
+		"mean_balanced_accuracy": 0.9532142857142858,
+		"stddev_balanced_accuracy": 0.03946136777261097,
+		"contamination": 0.0,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.9402976190476192,
+		"stddev_balanced_accuracy": 0.03857657226138218,
+		"contamination": 0.1,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.9340476190476192,
+		"stddev_balanced_accuracy": 0.029968686416539506,
+		"contamination": 0.2,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.9086309523809525,
+		"stddev_balanced_accuracy": 0.014918998981575312,
+		"contamination": 0.3,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.8177976190476192,
+		"stddev_balanced_accuracy": 0.03787061754803305,
+		"contamination": 0.4,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.6940476190476191,
+		"stddev_balanced_accuracy": 0.06650357388995012,
+		"contamination": 0.5,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.7007142857142855,
+		"stddev_balanced_accuracy": 0.052469351814355365,
+		"contamination": 0.6,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.6161309523809524,
+		"stddev_balanced_accuracy": 0.06440861268304963,
+		"contamination": 0.7,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.5648809523809523,
+		"stddev_balanced_accuracy": 0.07327281825951291,
+		"contamination": 0.8,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.4990476190476191,
+		"stddev_balanced_accuracy": 0.04683067632330287,
+		"contamination": 0.9,
+		"completeness": 1.0
+	},
+	{
+		"mean_balanced_accuracy": 0.48654761904761906,
+		"stddev_balanced_accuracy": 0.024593059876808907,
+		"contamination": 1.0,
+		"completeness": 1.0
+	}
+]
```

Comment:
I'm not so familiar with JSON, but POSIX standard would be to have a newline at EOF. I assume the JSON files are read/written by a library that just does not do that? Then it would probably not matter.

Reply:
yeah, written and read with `json`. But inserting a newline won't hurt, you are right.